docs/help.md

```diff
@@ -288,3 +289,16 @@ log.info({ msg: 'mapped to originalMsg' }, 'a message')
 // {"level":30,"time":1596313323106,"pid":63739,"hostname":"foo","msg":"no original message"}
 // {"level":30,"time":1596313323107,"pid":63739,"hostname":"foo","msg":"a message","originalMsg":"mapped to originalMsg"}
 ```
+
+<a id="best-performance-for-stdout"></a>
+## Best performance for logging to `stdout`
+
+The best performance for logging directly to stdout is achieved by using the
```

Comment:
```suggestion
The best performance for logging directly to stdout is _usually_ achieved by using the
```

Reply:
@mcollina just for the sake of curiosity could you elaborate on when the default configuration would not yield the best performance for logging directly to stdout?